docs/examples/1d_hist/1d_side_by_side.py

```diff
@@ -0,0 +1,83 @@
+"""
+1d side by side
+===============
+
+Plot multiple 1d histogram with categories side by side.
+
+"""
+
+###
+import boost_histogram as bh
+import numpy as np
+import matplotlib.pyplot as plt
+import plothist  # For the style
+
+np.random.seed(8311311)
+
+# Integer categories
+categories = ["A", "B", "C"]
+## Work also with integers
+# categories = [-5, 10, 137]
+
+# Axis with the 3 bins
+axis = bh.axis.StrCategory(categories=categories)
+
+# Generate data for 3 histograms
+data = [
+    np.random.choice(categories, 15),
+    np.random.choice(categories, 15),
+    np.random.choice(categories, 15),
+]
+
+# Create and fill the histograms
+histos = [bh.Histogram(axis, storage=bh.storage.Weight()) for _ in range(len(data))]
+histos = [histo.fill(data[i]) for i, histo in enumerate(histos)]
+
+# Convert the histograms to numpy arrays
+histo_arrays = [histo.values() for histo in histos]
```

Comment:
This is bad practice because we loose information in that operation. We should keep working with the boost_histogram objects.